### ISSUE TITLE
[nextalign] Install on MacOS

### ISSUE THREAD

**Issue by ivan-aksamentov** [nextstrain/nextclade#315]
See https://github.com/neherlab/nextalign/issues/24

**Comment by iskandr** [comment]
I was about to try out nextalign but saw this issue, curious if it's resolved. Also, any chance it's installable via either brew or bioconda?

**Comment by ivan-aksamentov** [comment]
Hi Alex @iskandr,

The issue mentioned was primarily about running *local* build on Mac, for development (development in docker was always available as an alternative). It was resolved, and one should be able to build Nextalign on Mac if needed., following the [developer's guide](https://github.com/nextstrain/nextclade/tree/master/packages/nextalign_cli#-developers-guide).

However, if you are just interested in using Nextalign, rather than developing it, it's much easier to just download the prebuilt binaries:
https://github.com/nextstrain/nextclade/releases

I am not sure it worth to spend effort on packaging, as Nextalign a static, self-contained executable, a single-file download. Users guide contains `curl` one-liners for quick installation:
https://github.com/nextstrain/nextclade/tree/master/packages/nextalign_cli#%EF%B8%8F-download-from-command-line

Will that work for you?

We might consider packaging (I created an issue https://github.com/nextstrain/nextclade/issues/357), but probably not at this time, as there are more important problems to resolve currently. However, contributions are very welcome!

I will close this as resolved, but feel free to open new issues if there are more questions, ideas, bugs to report etc.
